go-controller/pkg/ovn/base_network_controller_kubevirt.go

```diff
@@ -0,0 +1,63 @@
+package ovn
+
+import (
+	"fmt"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kubevirt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+)
+
+func (bnc *BaseNetworkController) ensureNetworkInfoForVM(origPod *corev1.Pod) error {
```

Comment:
I don't think I understand something; how does the caller know if this method is being called with the "original" pod ?

Reply:
You don't need the original pod you need another one that is not you since all the pods has the same NetworkInfo